### DISCUSSION TITLE
Super_L key combination not working

### DISCUSSION THREAD

**Discussion by qrockz** [i3/i3#5579] [Q&A]
Hi,

this works (_Super_L without another key_):
`bindsym Super_L exec rofi -modi "clipboard:greenclip print" -show clipboard -run-command '{cmd}'`

this **works not** (_Super_L+v_), why?
`bindsym Super_L+v exec rofi -modi "clipboard:greenclip print" -show clipboard -run-command '{cmd}'`

**Comment by daxadax** [comment]
same issue for me - not sure why this is

**Reply by orestisfl** [reply]
`Super_L` is a `keysym`, not a modifier. You need to use `Mod4`

**Comment by orestisfl** [comment]
`Super_L` is a `keysym`, not a modifier. You need `Mod4+v` instead
